src/wp-includes/html-api/class-wp-html-token.php

```diff
@@ -63,7 +63,7 @@ class WP_HTML_Token {
 	/**
 	 * Called when token is garbage-collected or otherwise destroyed.
 	 *
-	 * @var callable|null
+	 * @var ?callable
```

Comment:
Apparently PHPDoc doesn't actually support this nullable syntax, although PHP itself does. 

https://stackoverflow.com/a/50704179/93579

Same goes for below.

Reply:
Thanks for flagging, [[58742]](https://core.trac.wordpress.org/changeset/58742) was landed to fix PHPDoc annotations in many places.